els/categories/serializers.py

```diff
@@ -0,0 +1,8 @@
+from rest_framework import serializers
+from categories.models import category 
+
+# category Serializer
+class categorieserializer(serializers.ModelSerializer):
```

Comment:
Rename class name to `CategoriesSerializer` and use Pascal Casing
```suggestion
class CategoriesSerializer(serializers.ModelSerializer):
```